uber/model_checks.py

```diff
@@ -107,6 +110,12 @@ def total_cost_over_paid(attendee):
         return 'You have already paid ${}, you cannot reduce your extras below that.'.format(attendee.amount_paid)
 
 
+@prereg_validation.Attendee
+def reasonable_donation_amount(attendee):
+    if not (_valid_donation(attendee.amount_extra)):
+        return 'Donation amount must be a positive integer or zero.'
```

Comment:
I just noticed that although you check to make sure the number is under 1,000,000, you don't _say_ that it should be less than that in the error message. This can lead to confusion, since someone could enter 9999999999 and then assume that our system is broken because the error message they get doesn't tell them that the number they entered was too high.